site2/website/data/team.js

```diff
@@ -37,12 +37,24 @@ module.exports = {
       apacheId: 'guangning',
       roles: 'Committer'
     },
+    {
+      name: 'Hang Chen',
+      apacheId: 'chenhang',
+      org: 'Bigo Technology',
```

Comment:
```suggestion
      org: 'BIGO Technology',
```

![image](https://user-images.githubusercontent.com/48120384/97142276-4fbba780-179b-11eb-96ef-8e4a01c0f1cb.png)